parser/parser_pubsub.go

```diff
@@ -0,0 +1,79 @@
+package parser
+
+import (
+	"go/ast"
+	"reflect"
+	"strings"
+
+	"encr.dev/parser/est"
+	"encr.dev/pkg/identifiers"
+)
+
+const pubsubPackage = "encore.dev/pubsub"
```

Comment:
Not sure; I put it in the const so when @ekerfelt adds the runtime library this can be changed to match easily

Reply:
I agree; although `encore.dev/async/pubsub` would allow us to add other async constructs in and keep them grouped; however should cron's have been `async` too?